rovisor-home.html

```diff
@@ -0,0 +1,242 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="" />
+    <meta name="author" content="" />
+    <title>Rovisor Research</title>
+    <link href="styles.css" rel="stylesheet">
```

Comment:
style.css in the /css folder. Unless you provide the correct path, this file cannot be loaded in the browser.

Look for the similar issues in all the files and fix those.